src/app/search/directives/story-card.component.css

```diff
@@ -68,16 +68,7 @@ publish-image  {
   display: -webkit-flex;
   display: -ms-flexbox;
   display: flex;
-}
-.duration, .modified, .play-count {
-  width: calc(100%/3);
-}
-.play-count {
```

Comment:
This was hanging out for #56 but things will probably change by the time we have that info, so yeah no sense working around something we don't have yet.